src/main/java/com/iexec/blockchain/broker/BrokerService.java

```diff
@@ -40,6 +40,7 @@
 @Service
 public class BrokerService {
 
+    private final BrokerClient brokerClient;
     private final IexecHubService iexecHubService;
     private final ChainConfig chainConfig;
```

Comment:
This field seems to be unused now.

Reply:
The field is used in both constructor and `fireMatchOrders` method.

Reply:
It doesn't seem to be used anymore in `fireMatchOrders`:
https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/58/files#diff-246e39bcc274ccf419988a9e15b8b9bc0c5f20dd39a8afd16704a42e6ae4a5fbR100-R112

As it is only used in constructor, it has no reason to be stored as a field.

Reply:
https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/blob/bugfix/fix-potential-null-pointer-exception/src/main/java/com/iexec/blockchain/broker/BrokerService.java#L103

Reply:
We don't talk about the same field, my bad! `chainConfig` is the one unused, not `brokerClient`.

Reply:
fixed in ae9e4d2, sorry for the misunderstanding